src/utils/api/cmcApi.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @module utils/cmcApi
+ * @desc 
+ */
```

Comment:
It's a good move to keep your api calls encapsulated in their own domain because it'll make it less painful to add additional endpoint requests and keep them tight together. And also allow you to begin support instances where you may need to cycle api tokens. Right now you have a low volume request rate but if your site picked up 
I bet you'd get limited on twitter, reddit, and others and will need app tokens. To support this, your api clients would evolve into a class with a methods that often mirror the resource endpoints.

so you'd

```
import Api from 'utils/api';
const api = new Api('tokenx123')
api.get(params)
api.update(params)
//etc
```

Reply:
That makes a lot of sense, I hadn't considered that.